doc/source/user_guide/plotting_with_bokeh.md

```diff
@@ -0,0 +1,135 @@
+---
+jupytext:
+  text_representation:
+    extension: .md
+    format_name: myst
+kernelspec:
+  display_name: Python 3 (ipykernel)
+  language: python
+  name: python3
+---
+
+(plotting_with_bokeh)=
+
+# Plotting with Bokeh
+
+Arviz has the option to use {doc}`Bokeh <bokeh:index>` as backend which offers interactive plots. Although most of the functions in the {ref}`plot <plot_api>` module work seamlessly with any backend, some advanced plots may require the use of backend specific features. In this guide, advanced plotting with Bokeh will be covered.
+
+This page can be downloaded as a {jupyter-download:script}`Python script <plotting_with_bokeh>`
+or as a {jupyter-download:nb}`Jupyter notebook <plotting_with_bokeh>`.
+
+```{code-cell} ipython3
+import arviz as az
+import matplotlib.pyplot as plt
+import numpy as np
+
+az.style.use("arviz-darkgrid")
+
+# Confgure Bokeh as backend
+az.rcParams["plot.backend"] = "bokeh"
+az.output_notebook()
+```
+
+:::{note}
+To display the plots as output in a Jupyter notebook, it's required to call the function  {func}`arviz.output_notebook` beforehand.
+:::
+
+:::{warning} Page in construction
+:::
+
+## Customizing plots
```

Comment:
I think the level 2 headers here and in "combining ArviZ and bokeh plots" might be a bit confusing. Even if generated through arviz, all plots generated here are also bokeh plots, and customizing could also include adding further elements to the plot, not only using the arguments available in arviz functions.

Seeing the differences in the separation I think the titles could be: "Bokeh specific arguments" and "Extending ArviZ-Bokeh plots" or something of the sort. Then we should also go to the plots arguments guide and replace the `ax, backend, show` sections with two links to the bokeh and matplotlib specific sections.

That being said, the level 3 headings look fine, I would not change them to be the argument name only like in the plot argument guide. They should however have their own anchors though, like in the plot argument guide. The end goal is to then update the docstrings to have links to the right sections in the corresponding parameter description and keep the examples in the docstring to plot specific arguments and examples.

Reply:
I changed the name and level of the header. Once the docs are ready, we can add the links in `Plots’ arguments guide`.